doc/man.html

```diff
@@ -0,0 +1,211 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
```

Comment:
I was thinking of a specific target (e.g: doc).
It's more to have the command (and eventually options) somewhere to know how to do it next time we (especially I :-)) need it.
We can also imagine adding some check in travis (the continuous integration) to ensure that the HTML is up to date (e.g: `make doc && git status | grep html || fail`) but that's more a nice to have.